### DISCUSSION TITLE
GroupBy APIs [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2093] [Discord Threads]
> 🙋‍♂️ **farhanNoco**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### hey, yes, it will be coming back.. sorry it got missed out... cool that you were using it - we had not advertise it that much.. what was your usecase for groupby apis.. we'd initially written it for group by view  
#  
> 🙋‍♂️ **farhanNoco**  
#### We use the Noco API from our backend to serve a couple of frontend applications  
#  
> 💁 **o1lab**  
#### okok  
#  
> 💁 **o1lab**  
#### we will bring it in next to next release  
#  
> 🙋‍♂️ **farhanNoco**  
#### I also depend on the query parameters such as bt, bfields, hm, hfields which seems to be missing  
#  
> 💁 **o1lab**  
#### please refer to apis.nocodb.com  
#  
> 💁 **o1lab**  
#### these have changed  
#  
> 🙋‍♂️ **farhanNoco**  
#### Sorry to bother you again but I do not see a way. There are only fields, where and sort parameters for Table row list endpoint. Should I pass bt, hm, bfields, hfields inside these somehow?  
#  
> 💁 **o1lab**  
#### my bad - easier to check swagger generated (right top menu)  
#  
> 💁 **o1lab**  
#### semantics for bt, hm has changed  
#  
> 🙋‍♂️ **farhanNoco**  
#### Thanks! I tried to filter nested columns like this: http://localhost:8081/api/v1/db/data/noco/TEST/subscriptions?fields=cars&nested[cars][fields]=ConfigId   
#### But I still get all columns for cars instead of only ConfigId. Any idea why that could be?  
#  
> 👨‍💻 **pranavxc**  
#### <@973475031101349918>  May I know which version of Nocodb are you using ?  
#  
> 🙋‍♂️ **farhanNoco**  
#### 0.90.10  
#  
> 💁 **o1lab**  
#### relation is subscriptions has many cars ?  
#  
> 🙋‍♂️ **farhanNoco**  
#### no one car  
#  
> 💁 **o1lab**  
#### ya makes sense.. read it wrongly  
#  
> 👨‍💻 **pranavxc**  
#### Everything looks good..... can you try with encoded URL   
####    
#### `http://localhost:8081/api/v1/db/data/noco/TEST/subscriptions?fields=cars&nested%5Bcars%5D%5Bfields%5D=ConfigId`  
#  
> 🙋‍♂️ **farhanNoco**  
#### Thanks trying it now. Related question, is it expected that in 0.9.11 the query parameters are in PascalCase instead of snake_case? I see that the fields returned in JSON response are also in PascalCase in 0.9.11 (used to be snake_case in 0.9.10)  
#  
> 👨‍💻 **pranavxc**  
#### returned object keys will be column alias  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/973961343315099678)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
